llama_stack/apis/agents/openai_responses.py

```diff
@@ -116,6 +116,7 @@ class MCPListToolsTool(BaseModel):
     input_schema: dict[str, Any]
     name: str
     description: str | None = None
+    annotations: dict[str, Any] | None = None
```

Comment:
I think this is not just a `dict[str, Any]` but actually a sub-structure with some clearly specified keys.